extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -197,84 +192,124 @@ static public function newFromId( $id ) {
 	 * @return bool
 	 */
 	public function load( $master = false ) {
-		wfProfileIn( __METHOD__ );
-		$result = true;
```

Comment:
Got rid of `$result`.  It was only used to store a true/false return value for one return statement at the end (typically because the use of wfProfileIn|Out makes it annoying to have more than one return statement).  Returning immediately with `false` in places where `$result` was set to `false`.